docs/configuration.md

```diff
@@ -157,21 +157,7 @@ Define a path to an Open API V3 spec file. This can be a relative file path or a
 api_path: ./source/pets.yml
 ```
 
-## `owner_slack_workspace`
+## `owner_slack_workspace` and `default_owner_slack`
 
-The Slack workspace name used when linking to the Slack owner of a piece of
-content. If not provided, the owner of a piece of content (channel or user)
-won't be linked.
-
-```yaml
-owner_slack_workspace: gds
-```
-
-## `default_owner_slack`
-
-The default Slack user or channel name to show as the owner of a piece of
-content. Can be overridden using the `owner_slack` frontmatter config option.
-
-```yaml
-default_owner_slack: '#owner'
-```
+These attributes are used to specify the owner of a page.  See the separate
+documentation for page expiry for more details.
```

Comment:
can you make this a link?

Reply:
Linking is slightly awkward, because the docs are in this repo in a separate file, but pulled together into a single page at https://alphagov.github.io/tech-docs-manual/ I'll add a link to the rendered manual page for now (which is slightly chicken-and-egg...)